design.md

```diff
@@ -2,12 +2,26 @@
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
 
+We are targetting CS5 students with this design. We assume the user has a basic understanding of programming.
+
```

Comment:
So, CS5 students maybe a little into the semester? If I recall correctly, we did picobot at the very beginning so I'm not even sure if they have any grasp of programming at the point.